Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -233,7 +233,9 @@ private Fragment getFragmentByNavMenuItemId(int id) {
             case R.id.nav_utility:
                 fragment = UtilitiesFragment.newInstance();
                 break;
-
+                //Work In Progress Offline Maps
```

Comment:
this change shouldnt be a part of this commit

Reply:
Yeah, I was meaning to ignore this file change but it still got mixed in the commit.